KataTODOAPIClientTests/TODOAPIClientTests.swift

```diff
@@ -33,8 +33,9 @@ class TODOAPIClientTests: XCTestCase {
     func testSendsContentTypeHeader() {
         stub(condition: isMethodGET() &&
             isHost("jsonplaceholder.typicode.com") &&
+            hasHeaderNamed("Content-Type", value: "application/json") &&
             isPath("/todos")) { _ in
-                return fixture(filePath: "", status: 200, headers: ["Content-Type":"application/json"])
+                return fixture(filePath: "", status: 200, headers: ["Accept": "application/json"])
```

Comment:
What a weird change, what's the reasoning? I've been reading BothamNetworking changes and didn't see any justification.

Reply:
Especially in the return value, doesn't make much sense.

Reply:
That's the server response not the stub for the client request. I changed the header name to be able to explain that the fixture has not to be the same we use two lines above.